toolkit/templates/documents.html

```diff
@@ -1,16 +1,16 @@
 <ul class='document-list'>
   {% for item in items %}
     <li class='document-list-item'>
-      {% if item.file_type %}
-        <a href="{{ item.link }}" class='document-link-with-icon' {% if item.download %}download {% endif %}>
-          <span class='document-icon'>{{ item.file_type }}<span> document:</span></span>
+        <a href="{{ item.link }}" class='document-link{% if item.file_type %}-with-icon{% endif %}' 
+          {% if item.download %}download {% endif %}
+          {% if item.analytics %}data-analytics="{{ item.analytics }}" {% endif %}
+          {% if item.analytics_category %}data-analytics-category="{{ item.analytics_category }}" {% endif %}
+          {% if item.analytics_action %}data-analytics-action="{{ item.analytics_action }}" {% endif %}
+          {% if item.analytics_label %}data-analytics-label="{{ item.analytics_label }}" {% endif %}
+        >
+          {% if item.file_type %}<span class='document-icon'>{{ item.file_type }}<span> document:</span></span>{% endif %}
```

Comment:
Oh, I'd still like to get rid of the extra span, if we can

Reply:
<img width="526" alt="screen shot 2017-10-16 at 10 05 14" src="https://user-images.githubusercontent.com/4599889/31603905-8cb9272a-b259-11e7-95ce-b15c4bbe002f.png">

It's needed by css, if we get rid of it the icons don't work.